src/openrct2/peep/GuestPathfinding.cpp

```diff
@@ -1631,9 +1631,9 @@ Direction peep_pathfind_choose_direction(const TileCoordsXYZ& loc, Peep* peep)
  * @param y y coordinate of location
  * @return Index of gParkEntrance (or 0xFF if no park entrances exist).
  */
-static uint8_t get_nearest_park_entrance_index(uint16_t x, uint16_t y)
+static ParkEntranceIndex get_nearest_park_entrance_index(uint16_t x, uint16_t y)
```

Comment:
Its probably best to result the coordinates and not the index, that seems pointless :)